core/modules/layout/css/layout.admin.css

```
@@ -395,3 +395,25 @@
 [dir="rtl"] .edit-layout-top .dropbutton-wrapper {
   float: left;
 }
+
+details summary {
+  list-style-type: none;
+}
+details[open] summary span::before, details summary span::before {
+  content: "";
+  position: relative;
+  left: -.3em;
+  top: -.2em;
+  width: 0;
+  height: 0;
```

Are these two items necessary? I experimented with them and did not see that they are having any effect.
```suggestion
  width: 0;
  height: 0;
```